### PR TITLE
Add security manager subsystem tests coverage

### PR DESCRIPTION
**Pull request by zhantemirov**
Jira: [EAPQE-83](https://issues.redhat.com/browse/EAPQE-83)
Berg issue: #9 

Security Manager subsystem page contains "minimum permissions" and "maximum permissions" vertical navigation items, expecting two spec files to be created.

Resources to cover (from `/subsystem=securitymanager:read-resource(recursive=true)`):

- [X]  /subsystem=security-manager/deployment-permissions=default/minimum-permissions
- [X]  /subsystem=security-manager/deployment-permissions=default/minimum-permissions/actions
- [X]  /subsystem=security-manager/deployment-permissions=default/minimum-permissions/class
- [X]  /subsystem=security-manager/deployment-permissions=default/minimum-permissions/module
- [X]  /subsystem=security-manager/deployment-permissions=default/minimum-permissions/name
- [X]  /subsystem=security-manager/deployment-permissions=default/maximum-permissions
- [X]  /subsystem=security-manager/deployment-permissions=default/maximum-permissions/actions
- [X]  /subsystem=security-manager/deployment-permissions=default/maximum-permissions/class
- [X]  /subsystem=security-manager/deployment-permissions=default/maximum-permissions/module
- [X]  /subsystem=security-manager/deployment-permissions=default/maximum-permissions/name

### CODE REVIEW

File: packages/testsuite/cypress/e2e/security-manager/test-configuration-subsystem-security-manager-maximum-permissions.cy.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import { AddModuleCommandBuilder } from "@berg/commands";
+
+describe("TESTS: Configuration => Subsystem => Subsystem => Security Manager => Maximum Permissions", () => {
```
Comment (OndrejKotek) [inline packages/testsuite/cypress/e2e/security-manager/test-configuration-subsystem-security-manager-maximum-permissions.cy.ts:3]: > Subsystem => Subsystem

Extra Subsystem level?

File: packages/testsuite/cypress/e2e/security-manager/test-configuration-subsystem-security-manager-minimum-permissions.cy.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import { AddModuleCommandBuilder } from "@berg/commands";
+
+describe("TESTS: Configuration => Subsystem => Subsystem => Security Manager => Minimum Permissions", () => {
```
Comment (OndrejKotek) [inline packages/testsuite/cypress/e2e/security-manager/test-configuration-subsystem-security-manager-minimum-permissions.cy.ts:3]: > Subsystem => Subsystem

Extra Subsystem level?

File: packages/testsuite/cypress/e2e/security-manager/test-configuration-subsystem-security-manager-maximum-permissions.cy.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import { AddModuleCommandBuilder } from "@berg/commands";
```
Comment (OndrejKotek) [inline packages/testsuite/cypress/e2e/security-manager/test-configuration-subsystem-security-manager-maximum-permissions.cy.ts:1]: Is this needed?

File: packages/testsuite/cypress/e2e/security-manager/test-configuration-subsystem-security-manager-minimum-permissions.cy.ts [hunk 4]
```diff
@@ -0,0 +1,85 @@
+import { AddModuleCommandBuilder } from "@berg/commands";
```
Comment (OndrejKotek) [inline packages/testsuite/cypress/e2e/security-manager/test-configuration-subsystem-security-manager-minimum-permissions.cy.ts:1]: Is this needed?

File: packages/testsuite/cypress/e2e/security-manager/test-configuration-subsystem-security-manager-maximum-permissions.cy.ts (hunk 1 above)
Comment (zhantemirov) [reply packages/testsuite/cypress/e2e/security-manager/test-configuration-subsystem-security-manager-maximum-permissions.cy.ts]: Fixed, thank you

File: packages/testsuite/cypress/e2e/security-manager/test-configuration-subsystem-security-manager-minimum-permissions.cy.ts (hunk 2 above)
Comment (zhantemirov) [reply packages/testsuite/cypress/e2e/security-manager/test-configuration-subsystem-security-manager-minimum-permissions.cy.ts]: Fixed, thank you

File: packages/testsuite/cypress/e2e/security-manager/test-configuration-subsystem-security-manager-maximum-permissions.cy.ts (hunk 3 above)
Comment (zhantemirov) [reply packages/testsuite/cypress/e2e/security-manager/test-configuration-subsystem-security-manager-maximum-permissions.cy.ts]: Forgot to remove, it's fixed now

File: packages/testsuite/cypress/e2e/security-manager/test-configuration-subsystem-security-manager-minimum-permissions.cy.ts (hunk 4 above)
Comment (zhantemirov) [reply packages/testsuite/cypress/e2e/security-manager/test-configuration-subsystem-security-manager-minimum-permissions.cy.ts]: Forgot to remove, it's fixed now

**Comment by OndrejKotek** [comment]
@zhantemirov Could you add also other tests implemented in hal/testsuite.next please? See https://github.com/hal/testsuite.next/blob/master/tests-configuration-securitymanager/src/test/java/org/jboss/hal/testsuite/test/configuration/security_manager/SecurityManagerTest.java

**Comment by zhantemirov** [comment]
@OndrejKotek sure, didn't know about that. I'll come back with more tests later.

**Review by kstekovi — APPROVED** [review]
Thank you for you contribution. The coverage is same as in old TS and thanks for the new method.

**Pull request merged**